File: HackerRank/Python/Easy/E0008.py
# Problem Statement: https://www.hackerrank.com/challenges/nested-list/problem

N = int(input())
S = [[input(), float(input())] for _ in range(N)]
S = sorted(S, key=lambda s: (s[1], s[0]))
S = [s for s in S if s[1] != S[0][1]]
S = [s for s in S if s[1] == S[0][1]]
for s in S:	
	print(s[0])

File: HackerRank/C/Easy/E0008.c
/*
Problem Statement: https://www.hackerrank.com/challenges/sum-of-digits-of-a-five-digit-number/problem
*/

#include <stdio.h>

int sum_digits(int n) {
	int sum = 0;
	while (n != 0) {
		sum += n % 10;
		n /= 10;
	}
	return sum;
}

int main() {
	int n;
	scanf("%d", &n);
	printf("%d", sum_digits(n));
	return 0;
}